storage/range_gc_queue.go

```diff
@@ -145,6 +137,14 @@ func (q *rangeGCQueue) process(now proto.Timestamp, rng *Range) error {
 
 		// TODO(bdarnell): remove raft logs and other metadata (while leaving a
 		// tombstone). Add tests for GC of merged ranges.
+	} else {
+		// This range is a current member of the raft group. Acquire the lease
+		// to avoid processing this range again before the next inactivity threshold.
+		if err := rng.requestLeaderLease(now); err != nil {
+			if _, ok := err.(*proto.LeaseRejectedError); !ok {
+				log.Warningf("unable to acquire lease from valid range %s: %s", rng, err)
```

Comment:
I think this should be an info or maybe V(1). If we race against another node trying to get the lease, it doesn't matter who wins as long as someone does; any lease holder will prevent future GC attempts.